Mathlib/Probability/CondVar.lean

```diff
@@ -0,0 +1,194 @@
+/-
+Copyright (c) 2025 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.MeasureTheory.Function.ConditionalExpectation.Real
+import Mathlib.MeasureTheory.Integral.Average
+import Mathlib.Probability.Variance
+
+/-!
+# Conditional variance
+
+This file defines the variance of a real-valued random variable conditional to a sigma-algebra.
+
+## TODO
+
+Define the Lebesgue conditional variance (see GibbsMeasure for a definition of the Lebesgue
+conditional expectation).
```

Comment:
Readers who don't know the current ecosystem of repositories will have no idea of what this is referring to. Presumably you'll add that to Mathlib in a not too distant future anyway.
```suggestion
Define the Lebesgue conditional variance.
```

Reply:
I think it's useful for people to know that this is being worked on, to avoid duplicating effort. I've linked to the repo